packages/api/src/env.js

```diff
@@ -129,5 +130,17 @@ export function envAll (req, env, ctx) {
         secretAccessKey: env.S3_SECRET_ACCESS_KEY_ID
       }
     })
+    if (env.ENV === 'dev') {
+      // show me what s3 sdk is up to.
+      env.s3Client.middlewareStack.add(
+        (next, context) => async (args) => {
+          console.log('s3 request headers', args.request.headers)
+          return next(args)
+        },
+        {
+          step: 'finalizeRequest'
+        }
+      )
+    }
```

Comment:
This is not needed anymore right?

Reply:
This was left in to save others the pain of trying to figure out what the hell s3 is up to, as it took a while for me to figure out that i needed to add logging middleware to do it, so I figured it captures a bit of knowledge that we might benefit from again in the future.

Reply:
I think it is fine to add this as log debug level